src/components/RandomUsers/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react'
+import GetUsers from '../GetUsers'
+import Users from '../Users'
+import User from '../User'
+
+const RandomUsers = () => {
+    const [users, setUsers] = useState([])
+    const [user, setUser] = useState({})
+
+    return (
+        <>
+            <GetUsers controlUsersState={setUsers} />
+            <Users getUsers={users} controlUserState={setUser} />
+            <User getUser={user} />
```

Comment:
getUser sounds like a function name. Not an object
```suggestion
            <User user={user} />
```